cmd/parse/blocks/blocks.go

```diff
@@ -44,8 +45,17 @@ will be replaced with the data downloaded from the node.
 			end, _ := cmd.Flags().GetInt64(flagEnd)
 			force, _ := cmd.Flags().GetBool(flagForce)
 
+			lastDbBlockHeight, err := parseCtx.Database.GetLastBlockHeight()
+			if err != nil {
+				return err
+			}
+
 			// Get the start height, default to the config's height; use flagStart if set
 			startHeight := config.Cfg.Parser.StartHeight
+
+			getStartHeight := math.Max(float64(startHeight), float64(lastDbBlockHeight))
```

Comment:
Ups, my bad! I thought the method I suggested to use exited in the `math` package, but instead is in the Tendermint one. I think that in this case it would be better to define one method on our own and put it inside the `types` package: 

```go
func MaxInt64(a, b int64) int64 {
	if a > b {
		return a
	}
	return b
}
```
This way we don't have to do multiple type interference